_posts/2022-05-19-kotlin-context-receivers.markdown

```diff
@@ -0,0 +1,504 @@
+---
+layout: post
+title:  "Context Receivers in Kotlin"
+date:   2022-05-19 20:00:00 +0100
+categories: Kotlin
+author: Fatih Coşkun
+---
+# Context Receivers In Kotlin
+
+Kotlin [1.6.20](https://kotlinlang.org/docs/whatsnew1620.html) finally introduces 
+[context receivers](https://github.com/Kotlin/KEEP/blob/master/proposals/context-receivers.md#detailed-design), a new 
```

Comment:
Just making sure: did you want to link to the 'detailed design' chapter on purpose? When reading it I expected it linking to the very top of the document, something like this: https://github.com/Kotlin/KEEP/blob/master/proposals/context-receivers.md#context-receivers

Reply:
I will change it